services/throttle_definitions.proto

```diff
@@ -82,4 +82,6 @@ message ThrottleBucket {
  */
 message ThrottleDefinitions {
   repeated ThrottleBucket throttleBuckets = 1;
+  uint64 totalAllowedGasPerSecFrontend = 2;
+  uint64 totalAllowedGasPerSecConsensus = 3;
```

Comment:
This should be snake_case
```suggestion
  uint64 total_allowed_gas_per_sec_frontend = 2;
  uint64 total_allowed_gas_per_sec_consensus = 3;
```

Reply:
Moved these to global dynamic properties.